debezium-server/debezium-server-redis/src/main/java/io/debezium/server/redis/RedisStreamChangeConsumer.java

```diff
@@ -129,9 +159,13 @@ public void handleBatch(List<ChangeEvent<Object, Object>> records,
             throws InterruptedException {
         DelayStrategy delayStrategy = DelayStrategy.exponential(initialRetryDelay, maxRetryDelay);
 
-        LOGGER.info("Handling a batch of {} records", records.size());
+        LOGGER.trace("Handling a batch of {} records", records.size());
```

Comment:
I prefer us to remain this as INFO, see below.

Reply:
Same comment as before